src/SFA.DAS.Location.Domain/Interfaces/IPostcodeApiService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using SFA.DAS.Location.Domain.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Location.Infrastructure.ApiClient
+{
+    public interface IPostcodeApiService
+    {
+        Task<IEnumerable<SuggestedLocation>> GetAllStartingWithOutcode(string query, int resultCount = 10);
```

Comment:
Not sure we need a default here for the result count - its defaulted on the controller action as well